spec/system/admin/invoice_print_spec.rb

```diff
@@ -212,7 +212,9 @@
         end
 
         it "displays $0.00 when a line item has no tax" do
-          expect(page).to have_content "#{Spree::Product.first.name} (1g) 1 $0.00 $12.54"
+          expect(page).to have_content Spree::Product.first.name
+          expect(page).to have_content "(1g)"
+          expect(page).to have_content "1 $0.00 $12.54"
```

Comment:
I see Filipe did this for the same reason too. It's interesting though; Capybara is supposed to [normalise whitespace](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/system/support/capybara_setup.rb#L8-L11) to avoid this problem, so there must be something happening here.

I think the previous form is valuable, because it better asserts that the right details are next to each other. Imagine if there were other products, and the price from one product appeared next to the other. 
But it probably won't, so all good :D

Reply:
The problem is the table layout. The PDF reader looks at it visually, not logically. So even though the product name and `"(1g)"` are in the same table cell, the unit description is in the next line with a different font size and sometimes it's seen as new row and sometimes not. So sometimes we get `(1g)      1 $0.00 $12.54` and sometimes we get `1 $0.00 $12.54     (1g)`. It's a known issue of the PDF reader and there doesn't seem to be a good way forward.

Reply:
Ughh, that's no fun, thanks for explaining.